### PR TITLE
Publish details for CVE-2020-8555

### PR DESCRIPTION
**Pull request by tallclair**

### CODE REVIEW

File: 2020/8xxx/CVE-2020-8555.json [hunk 1]
```diff
@@ -1,18 +1,220 @@
 {
```
Comment (cve-team) [inline 2020/8xxx/CVE-2020-8555.json:1]: A language tag was used multiple times: description.

**Comment by joelsmith** [comment]
:+1: Looks good to me

File: 2020/8xxx/CVE-2020-8555.json [hunk 2]
```diff
@@ -1,18 +1,216 @@
...
+  "data_type": "CVE",
+  "data_format": "MITRE",
+  "data_version": "4.0",
+  "generator": {
+    "engine": "Vulnogram 0.0.9"
+  },
+  "CVE_data_meta": {
+    "ID": "CVE-2020-8555",
+    "ASSIGNER": "security@kubernetes.io",
+    "DATE_PUBLIC": "",
```
Comment (liggitt) [inline 2020/8xxx/CVE-2020-8555.json:11]: date?

File: 2020/8xxx/CVE-2020-8555.json [hunk 3]
```diff
@@ -1,18 +1,216 @@
...
+          "product": {
+            "product_data": [
+              {
+                "product_name": "Kubernetes",
+                "version": {
+                  "version_data": [
+                    {
+                      "version_name": "",
+                      "version_affected": "",
+                      "version_value": "prior to 1.15.12",
```
Comment (liggitt) [inline 2020/8xxx/CVE-2020-8555.json:37]: normally this is more like this:
```suggestion
                      "version_name": "",
                      "version_affected": "<",
                      "version_value": "1.15.12",
```

File: 2020/8xxx/CVE-2020-8555.json [hunk 4]
```diff
@@ -1,18 +1,216 @@
...
+        "lang": "eng",
+        "value": "The Kubernetes kube-controller-manager in versions v1.0-1.14, versions prior to v1.15.12, v1.16.9, v1.17.5, and version v1.18.0 are vulnerable to a Server Side Request Forgery (SSRF) that allows certain authorized users to leak up to 500 bytes of arbitrary information from unprotected endpoints within the master's host network (such as link-local or loopback services)."
+      }
+    ]
+  },
+  "references": {
+    "reference_data": [
+      {
+        "refsource": "CONFIRM",
+        "url": "https://github.com/kubernetes-security/security-disclosures/issues/207",
```
Comment (liggitt) [inline 2020/8xxx/CVE-2020-8555.json:175]: not accessible?

**Pull request merged**